fvm/src/call_manager/default.rs

```diff
@@ -294,7 +295,19 @@ where
             };
 
             // Make a store.
+            let gas_available = kernel.gas_available();
+            let fuel_to_add = match kernel.network_version() {
+                NetworkVersion::V14 | NetworkVersion::V15 => u64::MAX,
```

Comment:
Ouch, I don't like this one bit. Is it not possible to keep fuel metering on but not set a fuel limit (i.e. we meter fuel, but we never halt)?

Reply:
u64::MAX should be large enough to be impossible to reach by incrementing but I understand where you are coming from.

Reply:
Wasmtime actually only supports up to i64::MAX fuel. If you pass u64::MAX, it'll treat it as "infinity".

Reply:
Yeah, I figured it was fine based on [this](https://github.com/bytecodealliance/wasmtime/blob/main/crates/wasmtime/src/store.rs#L1290-L1296), and of course this is only relevant for testing on v14/15.